### ISSUE TITLE
Approximately how long does it take to complete a full training session on the RTX3090?

### ISSUE THREAD

**Issue by RXGuo** [jmliu206/LIC_TCM#26]
Approximately how long does it take to complete a full training session on the RTX3090?

**Comment by PuppetDiary** [comment]
On a training set with 50,000 images, I spent about 12 hours of training time with 4 v100 32G gpu's.

**Comment by studyingeugene** [comment]
@PuppetDiary  Wow I think it's fast. Can I ask your training details?
1. What optimizer you use 
2. Optimizer hyper parameters such as betas in Adam or weight decay factor
3. learning rate
4. Total batch per one step (batch per GPU * 4)
